docs/Troubleshooting/MySqlSizeIssue.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Maintenance Troubleshooting"
+sidebar_position: 4
+description: >
+  Maintenance Troubleshooting
+---
+
+## How to manage the quickly increasing MySQL disk consumption?
+
+DevLake is designed to collect data by first deleting the existing data and
```

Comment:
Can we add a sentence explaining why DevLake choose "delete + insert" instead of "update"?